primo-explore/custom/01NWU_INST-HSLV/css/custom1.css

```diff
@@ -1769,3 +1769,12 @@ prm-search-bar {
 prm-primary-bg.prm-hue1, prm-search-bar.prm-hue1, prm-spinner.prm-hue1.overlay-cover.light-on-dark:after, prm-topbar .top-nav-bar {
   background-color: #51468A;
 }
+
+/* homepage md cards */
+#searching-list, #sign-in-list {
+  margin: 0;
+}
+
+md-card.default-card md-card-title#get-help-title {
+  padding: 0px 15px;
```

Comment:
I'm not sure if it's easily done here, but I think in general it's good to use `em` or `rem` rather than `px` for scaling. If it's causing problems to use those don't worry about it though.

Reply:
Good idea, I am all for using generally better practices.